styles/style.css

```diff
@@ -1,176 +1,143 @@
 html {
```

Comment:
i am not sure what a lot of these styles are used for on the html.

top, right -> use margins instead

width, height -> not sure why you are constraining width and height for entire html element.

font-size is defined twice.

background-color shouldn't need !important?

also the line-height, letter spacing might be better off on more specific elements like p, h1, etc... or you might get unwanted side effects later.